app/components/project-post-list.js

```diff
@@ -1,25 +1,88 @@
 import Ember from 'ember';
 
+/**
+  `project-post-list` provides a list of posts within the project and allows filtering and paging.
+
+  ## Default usage
+  ```Handlebars
+    {{project-post-list posts=posts project=project}}
+  ```
+  ## Adding post filtering
+    These additional properties allow for filtering.
+  ```Handlebars
+    {{project-post-list
+      isFiltered=isFiltered
+      isFilteringClosedPosts=isFilteringClosedPosts
+      isFilteringOpenPosts=isFilteringOpenPosts
+      filterByStatus="filterByStatus"
+      filterByType="filterByType"
+      pageNumber=page
+      posts=model
+      project=project
+      removeTypeFilter="removeTypeFilter"
+      selectedTypes=selectedTypes
+    }}
+  ```
+
+  @class Component
+  @module project-post-list
+  @extends Ember.Component
+ */
+
 export default Ember.Component.extend({
   classNames: ['project-post-list'],
 
+  /**
+    @property store
+    @type Ember.Service
+   */
   store: Ember.inject.service(),
 
+  /**
+    Returns information about the posts in order to sort and display posts in
+    the control.
+
+    @property meta
+    @type
+   */
   meta: Ember.computed.alias('posts.meta'),
 
+  /**
+    Returns information about the posts to the `pager-control` to handle page
+    size.
+
+    @property options
+    @type Ember.Computed
```

Comment:
@rileytaylor `_normalizeMeta` returns an `Object`, if anything. But it is not an `Ember.Object`. The prototype is just `Object`, so that's what I would put the type as.